.github/workflows/docker-pub.yml

```diff
@@ -0,0 +1,45 @@
+name: Docker Image CI and CD
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+  
+    - name: Set up QEMU
```

Comment:
QEMU is for ?

Reply:
It is an add-on, incase we want to build multi arch images. Optional though!

Reply:
hmm, then for now lets not include it if it is not needed